tools/run_tests/run_xds_tests.py

```diff
@@ -257,7 +256,7 @@ def parse_port_range(port_arg):
 # sends an update with no localities when adding the MIG to the backend service
 # can race with the URL map patch.
 _TESTS_TO_FAIL_ON_RPC_FAILURE = [
-    'new_instance_group_receives_traffic', 'ping_pong', 'round_robin'
+    'ping_pong', 'round_robin'
```

Comment:
This can probably be all one line now (I think sanity complained)

Reply:
Done